README.md

```diff
@@ -170,9 +172,11 @@ object ExampleSparkJob {
       .setCheckpoint("Computations Started") // First checkpoint
     
     // A business logic of a spark job ...
+
+    // implicit FS is needed for setCheckpoint call, e.g. standard HDFS here:
```

Comment:
Isn't there a first `.setCheckpoint` on line 170 and this should be before it?

Reply:
Yes, it is, thank you!